js/components/playbackSpeedPanel.js

```diff
@@ -35,6 +35,11 @@ class PlaybackSpeedPanel extends React.Component {
   getPlaybackSpeedOptions() {
     // Only process speed options once per component mount
     if (!this.playbackSpeedOptions) {
+      if (this.props.skinConfig.playbackSpeed.length >= 1) {
+        // We take configured values from backlot
+        this.props.skinConfig.playbackSpeed.options =
+        JSON.parse(JSON.stringify(this.props.skinConfig.playbackSpeed));
```

Comment:
Honestly saying `JSON.parse(JSON.stringify(...))` is a sort of ugly construction, right? So when you need to clone the array, that has the only primitives inside, you can use more simple and beautiful spread operator: `this.props.skinConfig.playbackSpeed.options = [... this.props.skinConfig.playbackSpeed]`